config/routes.rb

```diff
@@ -1,15 +1,20 @@
 require 'sidekiq/web'
 Rails.application.routes.draw do
+
   mount Attachinary::Engine => "/attachinary"
 
   get 'story/new'
-  get 'locales' => 'application#locales'
   get 'projects/archived' => 'projects#archived'
+  put 'locales' => 'locales#update', as: :locales
+
+  get 't/:id' => 'teams#switch', as: :teams_switch
+  resources :teams
 
   resources :projects do
     member do
       get :import
       patch :import_upload
```

Comment:
I was thinking the same but you do have many teams (create is not restricted to just one).  But an admin can only manage his own teamteam.